components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -599,7 +600,26 @@ func TestDocuments_ValidatePackage(t *testing.T) {
 
 				return []*open_resource_discovery.Document{doc}
 			},
-		}, {
+		},
+		{
+			Name: "Not incremented `version` field when Package",
+			DocumentProvider: func() []*open_resource_discovery.Document {
+				doc := fixORDDocument()
+				doc.Packages[0].Industry = json.RawMessage(`["Mining"]`)
+				doc.Packages[0].Version = fixPackagesWithLowerVersion()[0].Version
```

Comment:
No need to lower the version value here and in all similar test cases (for Apis/Events).

For further explanation check the comment in the `open_resource_discovery/fixtures_test.go`